sensor/common/metrics/metrics.go

```diff
@@ -128,12 +129,12 @@ var (
 		Help:      "A counter of the total number of processes sent to Central by Sensor",
 	})
 
-	totalProcessesReceivedCounter = prometheus.NewCounter(prometheus.CounterOpts{
+	totalProcessesCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I'm not sure if we should add another metric for the new service. AFAIC this metric tracks messages received from the old service right?

Reply:
Yeah, this one is tracking the ones from the legacy service, I'll add another for the new one.

Reply:
Ended up doing something slightly different, because the legacy channel is more tied to the concept of "signals", I renamed this metric `totalProcessesSignalCounter` and `totalProcessesCounter` now tracks the queue from the new service. Hopefully this isn't too confusing.